projects/aca-shared/src/lib/components/open-in-app/open-in-app.component.ts

```diff
@@ -51,4 +52,10 @@ export class OpenInAppComponent {
   openInApp(): void {
     this.window.location.href = this.redirectUrl;
   }
+
+  onCloseDialog(): void {
```

Comment:
what about clicking outside the dialog? it will close it without calling this event

Reply:
If user will click outside dialog will not close as this is the requirement until and unless user close the dialog with close icon.